### ISSUE TITLE
Latest updates 

### ISSUE THREAD

**Issue by gedw99** [mlctrez/goapp-mdc#2]
hey 👋 

Google have now built out alot more good stuff for v2 so worth trying it out and if we can update this.

they still have some components not done in v2. Data picker etc.

—-

The other bit - yes I know it should be in another issue but anyways ..

I am really liking go-app and I think it’s possible to integrate with some of the OS level autofill too. To get an idea of what I mean:
- login user name using .well-known standards
- Address using the same .well-known
- Credit card usingbssne …

these autofill make things smooth for a user.

I have gotten sone of these auto gills working and so would like to Par these to this repo.

**Comment by mlctrez** [comment]
@gedw99  This was an experiment that I took on at the time to learn a bit more about starting an open source library and mainly to learn something more than a go-app "hello world".

There are probably many things that I would do differently if I wrote this today based on what I've learned. Feel free to fork this and test out MDC V2 updates. I don't think it would be to difficult to adapt what changes have occurred to this codebase.

>  integrate with some of the OS level autofill 

I'm not sure what you mean here?
